spec/spec_helper.rb

```diff
@@ -1,6 +1,13 @@
 require "simplecov"
 SimpleCov.start
 
+require "vcr"
+VCR.configure do |c|
+  c.hook_into :faraday
+  c.cassette_library_dir = "spec/support/vcr_cassettes"
+  c.default_cassette_options = {:record => :new_episodes}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.